### DISCUSSION TITLE
Homepage's All Activity is not dark

### DISCUSSION THREAD

**Discussion by ben-sommer** [community/community#22214] [Accessibility]
<p>Dark mode looking sick :fire: I noticed that the ‘All Activity’ section on the home page is still in light mode though.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/ben-sommer">@ben-sommer</a>, we are happy you like it! You can set it to be with your system or to always be dark or light:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/0/a02f0c07efce6290b0b8d5388a0022ecc6870505.png" data-download-href="/uploads/short-url/mR3bXSnEFNYyEUfhPQ01UMg3QIB.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/20666190/181069565-bf570a4e-15c7-464d-b778-862d85a705df.png" alt="image" data-base62-sha1="mR3bXSnEFNYyEUfhPQ01UMg3QIB" width="690" height="270" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a02f0c07efce6290b0b8d5388a0022ecc6870505_2_690x270.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a02f0c07efce6290b0b8d5388a0022ecc6870505_2_1035x405.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a02f0c07efce6290b0b8d5388a0022ecc6870505_2_1380x540.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a02f0c07efce6290b0b8d5388a0022ecc6870505_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1552×608 36.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ben-sommer** [comment]
<p>I set it to dark mode but I noticed that only some of the sections were still in light mode. Might have been some browser extensions - will check now.</p>

**Comment by ben-sommer** [comment, marked as answer]
<p>Seems to have been one of my browser extensions, or it just got fixed now. Either way, this issue is now resolved.</p>

**Comment by smac89** [comment]
<p>Yup I was just about to comment that you probably have <code>github-refined</code> enabled.</p>
<p>Looks like it’s already been reported as well:<br>
</p><aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/sindresorhus/refined-github/issues/3798" target="_blank" rel="noopener nofollow ugc">github.com/sindresorhus/refined-github</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/sindresorhus/refined-github/issues/3798" target="_blank" rel="noopener nofollow ugc">Support for Github Darkmode</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2020-12-08" data-time="17:36:49" data-timezone="UTC">05:36PM - 08 Dec 20 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/fionera" target="_blank" rel="noopener nofollow ugc">
          <img alt="fionera" src="https://user-images.githubusercontent.com/8305511/181069568-aaae9058-19d6-4ad7-b12e-bf04fe307630.jpeg" class="onebox-avatar-inline" width="20" height="20">
          fionera
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">When using Refined Github with the new Github Darkmode the Activity List is rendered with a white background.
https://imgur.com/a/uy8K50q</p>
</div>

<div class="labels">
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">Please! ♥︎</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">bug</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">help wanted</span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by ben-sommer** [comment]
<p>Ah, good spot. I was looking at my Github Dark Mode extension instead :man_facepalming:</p>

**Comment by iambrj** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/5/2547aa15f24e88853a2a9749f1ef8cf2276f6fe5.png" data-download-href="/uploads/short-url/5jNcOrdwiGgtwekrtsVxHoZfoJ7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29101670/181069574-859984e8-693d-4ab7-b10f-f67071f72ea5.png" alt="image" data-base62-sha1="5jNcOrdwiGgtwekrtsVxHoZfoJ7" width="690" height="375" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/5/2547aa15f24e88853a2a9749f1ef8cf2276f6fe5_2_690x375.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/5/2547aa15f24e88853a2a9749f1ef8cf2276f6fe5_2_1035x562.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/5/2547aa15f24e88853a2a9749f1ef8cf2276f6fe5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/5/2547aa15f24e88853a2a9749f1ef8cf2276f6fe5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1358×740 137 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/iambrj">@iambrj</a>! I’m going to refer you to <a href="https://github.community/t/homepages-all-activity-is-not-dark/147810/5">this related topic</a>. If you are using the <code>github-refined</code> browser extension enabled, the “All activity” feed will be broken.</p>

**Comment by mohsinulhaq** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/1/61991c9e544829105df5e45148632a56330a7ebc.png" data-download-href="/uploads/short-url/dVom183OSn2hkuiMveiJwLt3nBi.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7688231/181069575-d6043715-f2df-48a5-90f6-43996f7ae568.png" alt="image" data-base62-sha1="dVom183OSn2hkuiMveiJwLt3nBi" width="690" height="363" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61991c9e544829105df5e45148632a56330a7ebc_2_690x363.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61991c9e544829105df5e45148632a56330a7ebc_2_1035x544.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61991c9e544829105df5e45148632a56330a7ebc_2_1380x726.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61991c9e544829105df5e45148632a56330a7ebc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2740×1444 449 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mohsinulhaq** [comment]
<p>It’s now fixed! Thanks.</p>

**Comment by gaknoll** [comment]
<p>Glad it’s now fixed <a class="mention" href="/mohsinulhaq">@mohsinulhaq</a> - this is caused by having the <code>refined-github</code> extension enabled. Hope you’re enjoying dark mode!</p>

**Comment by cehfisher** [comment]
Hi @queenofcorgis - this stale issue is resolved [per this comment by the original poster](https://github.com/orgs/community/discussions/22214#discussioncomment-3235934). Could you please close this issue?
